akka-docs/src/main/paradox/typed/from-classic.md

```diff
@@ -145,11 +146,11 @@ understand it.
 when creating an `ActorSystem` in Typed you give it a `Behavior` that will be used as the top level actor, also known 
 as the user guardian.
 
-It's from the user guardian you create additional actors for the application and initialize tools like
-Cluster Sharding. In contrast, such initialization are typically performed from the "outside" after
-starting a classic `ActorSystem`.
+Additional actors for an application can be created from the user guardian alongside performing the initialization
```

Comment:
I think "can be created" sounds too optional, this changes the original meaning which is more of a "this is how you do it".

Reply:
So instead of `Additional actors for an application can be created`, you would change this to `Additional actors for an application are created`, correct?

Reply:
That is better 👍

Reply:
Done.